src/main/java/de/cyklon/reflection/entities/ReflectClass.java

```diff
@@ -33,7 +34,7 @@ static <D> ReflectClass<D> wrap(@NotNull Type type) {
 
 	@NotNull
 	@SuppressWarnings("unchecked")
-	static <D> ReflectClass<D> getClass(@NotNull D obj) {
+	static <D> ReflectClass<D> getSubclass(@NotNull D obj) {
```

Comment:
Why have you renamed this method? This method only returns the class of the specified object as a ReflectClass.